### PR TITLE
Adding "BEST: BERT Pre-training for Sign Language Recognition with Coupling Tokenization"

### PR DESCRIPTION
**Pull request by cleong110**
My notes on the process: https://github.com/cleong110/sign-language-processing.github.io/issues/19

A few things:
* No I don't know what BEST stands for, it's not called out explicitly in the paper but I _believe_ it is **B**ERT pre-training for **S**ign language recognition with coupling **T**okenization
* Might be good to double-check some of the stuff about coupling tokenization, I might not have understood it properly
* extremely similar in a number of ways to #54, to the point where I could not resist referencing it, and I think we need to merge that one first. Similar approach, evaluated on the same datasets with similar results. Even the table layouts are similar, with both of them calling their method "Ours +R" in the "from video" results. Neither one offers any code. 

TODO: 
- [ ]  rewrite for conciseness
- [ ] Merge #54 first

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -961,6 +961,19 @@ They found that for both forms of fingerspelling, on average, the longer the wor
 Furthermore, they found that less time is spent on middle letters on average, and the last letter is held on average for longer than the other letters in the word.
 Finally, they used this information to construct an animation system using letter pose interpolation and controlled the timing using a data-driven statistical model.
 
+### Pretraining and Representation-Learning
+
+<!-- BEST seems to be **B**ERT pre-training for **S**ign language recognition with coupling **T**okenization -->
+@Zhao2023BESTPretrainingSignLanguageRecognition introduce BEST, a pretraining method based on masked modeling of pose sequences using a coupled tokenization scheme.
+The method takes in pose triplet units (left hand, right hand, and upper-body with arms) as inputs.
+The pose for each part of the triplet is tokenized into discrete codes [@van_den_Oord_2017NeuralDiscreteRepresentationLearning].
```
Comment (AmitMY) [inline src/index.md:969]: merge these two sentences
```
This method takes pose triplet units (left hand, right hand, and upper-body with arms) as inputs, each tokenized into discrete codes [@van_den_Oord_2017NeuralDiscreteRepresentationLearning].
```

File: src/index.md [hunk 2]
```diff
@@ -961,6 +961,19 @@ They found that for both forms of fingerspelling, on average, the longer the wor
...
+Then masked modeling is employed: any or all of the three parts may be masked, e.g. left hand, or right hand, or body+hand, or all of them...
```
Comment (AmitMY) [inline src/index.md:970]: then, comma
e.g., comma

this sentence is not very readable, and `...` (3 dots) is not common in academic writing

File: src/index.md [hunk 3]
```diff
@@ -961,6 +961,19 @@ They found that for both forms of fingerspelling, on average, the longer the wor
...
+Unlike @hu2023SignBertPlus, they do not mask multi-frame sequnces ("clips") or sub-frame portions of a pose unit (joints).
```
Comment (AmitMY) [inline src/index.md:971]: they do mask joints - just groups of joints

File: src/index.md [hunk 4]
```diff
@@ -961,6 +961,19 @@ They found that for both forms of fingerspelling, on average, the longer the wor
...
+They validate their pretraining method isolated ISR (MS-ASL [@dataset:joze2018ms], WLASL [@dataset:li2020word], SLR500 [@huang2019attention3DCNNsSLR] and NMFs-CSL [@hu2021NMFAwareSLR]).
```
Comment (AmitMY) [inline src/index.md:972]: missing "on" or "using"

File: src/index.md [hunk 5]
```diff
@@ -961,6 +961,19 @@ They found that for both forms of fingerspelling, on average, the longer the wor
...
+They experiment with both pose-to-gloss and video-to-gloss via fusion with I3D [@carreira2017quo].
+Results on these datasets are SOTA compared to previous methods, and quite similar to those of SignBERT+ [@hu2023SignBertPlus]
```
Comment (AmitMY) [inline src/index.md:974]: missing dot at end of sentence.

generally, a chatgpt rewrite can help this PR, and ideally all PRs should have such a rewrite before I look at them

File: src/references.bib [hunk 6]
```diff
@@ -3150,3 +3150,30 @@ @inproceedings{sellam-etal-2020-bleurt
  url = {https://aclanthology.org/2020.acl-main.704},
  year = {2020}
 }
+
+@article{Zhao2023BESTPretrainingSignLanguageRecognition,
+  title        = {BEST: BERT Pre-training for Sign Language Recognition with Coupling Tokenization},
```
Comment (AmitMY) [inline src/references.bib:3155]: {BEST} {BERT} {S}ign {T}okenization

**Review by AmitMY — CHANGES_REQUESTED** [review]

**Comment by cleong110** [comment]
Merged latest, working on rewrites

File: src/index.md (hunk 5 above)
Comment (cleong110) [reply src/index.md:970]: Yeah, meant to do so, got interrupted, just now getting to it. I think I'll do it before even making a PR next time.

Comment (cleong110) [reply src/index.md:970]: Wow this is riddled with issues upon reread, typos and more. Apologies.

File: src/index.md (hunk 3 above)
Comment (cleong110) [reply src/index.md:967]: Ah, right. E.g. one of the hands. But not one of the fingers.

**Comment by cleong110** [comment]
All right did some rewrites. https://chatgpt.com/share/f42199e8-bd03-4ec3-9913-e5e832bb2885 had various suggestions, some of which I incorporated. It consistently struggles with citations that start with `[dataset:`, and will sometimes write out the rendered citation, and has a few other issues, but generally seems helpful

**Review by AmitMY — APPROVED** [review]
